lib/cavalcade-runner-to-cloudwatch/plugin.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+namespace HM\Cavalcade\Runner\CloudWatch;
+
+use HM\Cavalcade\Runner\Hooks;
+use HM\Cavalcade\Runner\Runner;
+use HM\Cavalcade\Runner\Worker;
+use HM\Cavalcade\Runner\Job;
```

Comment:
Could potentially `use HM\Cavalcade\Runner\{Hooks, Job, Runner, Worker}` or just `use` the namespace?